mantidproject/mantid
# Convolution fit sequential algorithm

Fixes #12419 

This replaces the python algorithm ConvFit (in IndirectDataAnalysis.py). This is really only ever run from the ConvFit Tab and is a workflow algorithm with the main task being a sequential call to PlotPeakByLogValue.

ToDo: 
- [x] .rst
- [x] release notes - http://www.mantidproject.org/ReleaseNotes_3_5_Framework_Changes#New
# To Test:
- Open the ConvFit Tab (Interfaces>Indirect>Data Analysis> ConvFit)
- Load relevant files using the ConvFIt GUI 
  - I used `irs26172_graphite002_red` and `irs26173_graphite002_res`
- Select one of the FitType from the drop down menu and then select Run from the bottom of the tab
- This should produce several workspaces in the ADS:
  - `_Results` MatrixWorkspace
  - `_Parameters` TableWorkspace
  - `_Workspaces` WorkspaceGroup
- Ensure these Workspaces along with the original `_res` and `_red` files are the only Workspaces in the ADS
- Ensure the data in the Workspaces matches the expected values
  - Open up version of Mantid **NOT** made from this PR
  - Run the same process as above with the same files
  - Compare the results from each run 
    - They should be identical
- In order to ensure correct functionality of this algorithm it may be advised to check the values for different runs that vary with:
  - Delta Function
  - Fit Type
  - Temperature
  - Initial values
